README.md

```diff
@@ -1,14 +1,144 @@
+
+ * In the 'General' tab:
+
+    * Set the public-facing name.
+   * In the ['Users & Permissions'](https://console.firebase.google.com/u/2/project/`example-project`/settings/iam) tab:
+
+    * Add any other users who will need "Owner" level access to this project (e.g. a backup owner who can change things if needed, deployments).
+    * Add the users who will need "Editor" level access to this project.
+
+3. Optionally, upgrade your account. Click the gear icon next to _Project Overview_ again, and then select ['Usage and billing'](https://console.firebase.google.com/u/2/project/`example-project`/usage), then select the Blaze plan.
+
+### Step 3: Add a New Web App to the Firebase Project
+
+1. Click on _Project Overview_, then again on [_Project Settings_](https://console.firebase.google.com/u/2/project/`example-project`/settings/general).
+
+ * In the 'General' tab:
+
+    * Go to the 'Your Apps' section at the bottom.
+    * Click on the `</>` button to create a new web app
+    * Add firebase to your web app
+    * Choose an app nickname (eg. `example-project`)
+    * Tick 'Also set up Firebase Hosting for this app', leaving the dropdown that appears on the default setting
+    * Click _Register app_
+    * Wait for the _loading_ spinner to finish
+    * Click _Next_ to skip past this screen
+    * Click _Next_ to skip past this screen as well
+    * Finally, you can click _Continue to the console_ to skip past this last screen; internal deployments are handled automatically through our CircleCI workflow, so we don't need to deploy manually
+
+This part of the setup is now complete!
+
+
+### Step 4: Set up Firebase Hosting
+
+1. From the Firebase console, within the appropriate project, click on ['Hosting'](https://console.firebase.google.com/u/2/project/`example-project`/hosting) on the left hand menu.
+
+2. Click on _Get started_.
+
+ * You can skip through these next screens by clicking _Next_
+  * Click _Continue to the console_
+
+2. OPTIONAL Add a custom domain.
+
+ * In the hosting dashboard, click _Add custom domain_
+ * Enter the details for the domain to use (eg. `example.sparkle.space`)
+    **Note:** you will also need to configure the DNS to make this work properly (see below), and this may take 24-48hrs to take effect
+ * Don't check 'Redirect `example.sparkle.space` to an existing website'
+ * Click _Continue_
+ * If you haven't verified your domain before, you'll need to do this here. You will likely need to create a DNS 'TXT' record to prove that you own the domain, containing a `google-site-verification` value
+ * You can follow similar steps to the ones described below to do this: _Enter domain --> Verify ownership --> Go live_
+ * You will need to enter the provided DNS 'A' records into your domain registrar (eg. GoDaddy) to point your domain at the Firebase servers (record type, host, value)
+
+You can close this page and return to the Firebase console.
+
+</details>
+
```

Comment:
can't find where we open this tag.
```suggestion
```

Reply:
I took out DNS settings section. We may want to include in a sub-doc later but it seemed unnecessary for just getting started. Was going to include the GoDaddy example as a dropdown hence the <details> html but that is a relic, good catch!